frontend/src/queries/schema.ts

```diff
@@ -21,6 +21,18 @@ import {
     LifecycleToggle,
 } from '~/types'
 
+/**
+ * PostHog Query Schema definition.
+ *
+ * This file acts as the source of truth for:
+ *
+ * - frontend/src/queries/schema.json
+ *   - generated from typescript via "pnpm run generate:schema:json"
```

Comment:
If `schema.json` and `schema.py` are only generated from this file, do we need to commit them? I can see this being a bit easier, but I'm a bit worried about them getting of out of sync as it's possible to commit changes to the wrong file. Plus they're really big. Could be compiled in package.json's `prepare`

Reply:
Says the guy who just committed gigabytes of PNGs to the repo :D.

Just like snapshots, I'm afraid if we don't commit them to git, we won't see any unexpected changes in the schema. Since this is a very important schema that'll be used to generate public facing docs, tooling for users, etc, I'd prefer to not take any chances. 

It also makes development easier, if the files are already there. Unrelated, I've also been playing around with a Kea typegen [inline types mode](https://github.com/keajs/kea-typegen/pull/40) that would also commit the types into the repo... for clean, fast and reproducible builds.

Also, I still want to add a github action that throws a wrench when the schema is out of sync.

Reply:
Having the kea types ship with git would help a lot for reviews etc. where switching branches takes a bit of time at the moment, so +1 for a solution there.

Reply:
Isn't `schema.ts` the snapshot in this case? The derived `.json` and `.py` versions ones are just distractions in a code review.
But fair enough, should be okay IF we have a wrench-throwing action.